digital-twins/jsonld-contexts/observed_attributes.jsonld

```diff
@@ -0,0 +1,9 @@
+  {
+    "@context": {
+        "name": "https://uri.fiware.org/ns/data-models#name",
+        "description": "https://uri.fiware.org/ns/data-models#description",
+        "blueprint": "https://uri.fiware.org/ns/data-models#blueprint",
```

Comment:
I think the `uri.fiware.org` namespace is no longer used. you can use the smart data models one instead.